### ISSUE TITLE
Add all commands to the bot help message

### ISSUE THREAD

**Issue by rachelmcr** [Automattic/wp-telegram-bot#21]
The bot's help message only lists one possible command:

`Type 'follow https://yourexcellentsite.com' into the channel or group`

I discovered that I could use the command `unfollow https://yourexcellentsite.com` to unfollow a site, but then I started wondering if there were other hidden commands I didn't know about.

It would be nice to have a list in the help message of all possible commands, so there's no doubt about what's possible.

**Comment by markryall** [comment]
Yes - I agree with @hoverduck that we should replace the current commands with slash commands and make the bot respond to `/start` or `/help` in a more conventional way.

This would address all commands (and open the possibility of adding more later).
